src/devices/Gpio/README.md

```diff
@@ -0,0 +1,79 @@
+# Sunxi GPIO Driver for .NET
+
+**sunxi** represents the family of ARM SoCs from Allwinner Technology. This project contains a **full function(PULL-UP, PULL-DOWN)** generic GPIO driver `SunxiDriver` for Allwinner SoCs and some special GPIO drivers like `OrangePiZeroDriver`, `OrangePiLiteDriver`, `OrangePiLite2Driver`.
+
+## Getting started
+
+### Generic GPIO driver: `SunxiDriver`
+
+```C#
+// Beacuse this is a generic driver, the pin scheme can only be Logical.
+// The base addresses can be found in the corresponding SoC datasheet.
+using GpioController gpio = new GpioController(PinNumberingScheme.Logical, new SunxiDriver(cpuxPortBaseAddress: 0x01C20800, cpusPortBaseAddress: 0x01F02C00));
```

Comment:
I think the sample shouldn't start with the most complex option, start with options where passing in the address is not necessary